.github/workflows/cypress_tests.yml

```diff
@@ -102,6 +102,7 @@ jobs:
         run: |
           pip install boto3
           npm ci --ignore-scripts .
+          cypress install
```

Comment:
I think you need either `npx cypress install` or `./node_modules/.bin/cypress install`